docs/sources/installation/debian.md

```diff
@@ -24,6 +24,7 @@ installation.
 
 ## Install Stable
 
+See [https://grafana.com/grafana/download?platform=linux](https://grafana.com/grafana/download?platform=linux) for the current download and installation instructions.
```

Comment:
We have this in table above this paragraph, maybe we should have a section heading named 

### Download links

To make the table & download page links more visible?

Reply:
The reason I opened this issue is that I could did not find it until I changed my search terms and finally got to the download page -- that has the real content I want when I search google for:
"install grafana ubuntu"

I get:
![image](https://user-images.githubusercontent.com/705951/61079651-1c9faa00-a3d8-11e9-9d6c-2b06fb1a0f7c.png)

Then I open it up and see:
<img src="https://user-images.githubusercontent.com/705951/61079577-072a8000-a3d8-11e9-8f36-fc2d3c27d507.png">

it took me a while to find the actual version numbers I could use in the install link

Reply:
But the download page has the full links and commands. So I think it would be better to make the link in the table easier to find instead of adding a new link?

Reply:
I am not questioning the root reason for the change, just the solution of adding a new differently named link right below the existing link instead of renaming or making the extent link more clearer / noticeable

Reply:
yes -- totally agree, and not sure what the best solution is :(

Maybe removing the example part entirely?  and say "See to http://....  for detailed instructions"?  or something like that.

The right answer for "install grafana ubuntu" is exactly what is on the download page!
![image](https://user-images.githubusercontent.com/705951/61083214-0b5a9b80-a3e0-11e9-8f0d-7f44afdda020.png)